I/IOAPI/build_tarballs.jl

```diff
@@ -0,0 +1,131 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "IOAPI"
+version = v"3.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/cjcoats/ioapi-3.2.git", "ef5d5f4e112c249b593b19426421f25d79ae094b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+apk add tcsh # build script is in csh
+
+cd $WORKSPACE/srcdir/ioapi-3.2/ioapi/
+export HOME=${WORKSPACE}/srcdir
+export BIN=tmp
+export BINDIR=${HOME}/ioapi-3.2/${BIN}
+mkdir $BINDIR
+
+sed -i 's/-ffast-math /-fPIC /g' Makeinclude* # Get rid of forbidden -ffast-math flag
+sed -i 's/-DNEED_ARGS=1/-DNEED_ARGS=1 -DIOAPI_NCF4=1/g' Makeinclude* # Specify NetCDF v4
+
+if [[ "${target}" == *i686* ]]; then
+    sed -i 's/-m64/-m32/g' Makeinclude* # Specify 32-bit build
+elif [[ "${target}" == *aarch64* ]]; then
+    sed -i 's/-m64//g' Makeinclude* # Get rid of x86 flag
+fi
+
+cp Makefile.nocpl Makefile
+cp Makeinclude.Linux2_x86_64gfort Makeinclude.$BIN
+
+make fixed_src
+make # Parallel make (-j > 1) does not work
+
+cp fixed_src/* ${includedir} # FORTRAN .EXT (include) files
+
+gfortran -shared -fPIC -fopenmp -o ${libdir}/libioapi.${dlext} -Wl,$(flagon --whole-archive) ${BINDIR}/libioapi.a -Wl,$(flagon --no-whole-archive) -lnetcdf -Wl,$(flagon --no-whole-archive) -lnetcdff
```

Comment:
Try this (the `flagon` script has a bug on macOS at the moment :persevere:)
```suggestion
gfortran -shared -fPIC -fopenmp -o ${libdir}/libioapi.${dlext} -Wl,$(flagon --whole-archive) ${BINDIR}/libioapi.a -Wl,$(flagon --no-whole-archive | cut -d' ' -f1) -lnetcdf -lnetcdff
```